src/Geno/C++/GUI/MainWindow.h

```diff
@@ -89,6 +91,12 @@ class MainWindow
 	OutputWindow*      pOutputWindow      = nullptr;
 	FindInWorkspace*   pFindInWorkspace   = nullptr;
 
+//////////////////////////////////////////////////////////////////////////
+
+	WorkspaceVector&      GetRecentWorkspaces()     {                                 return m_RecentWorkspaces;      }
+	std::filesystem::path GetMostRecentWorkspace()  { if( m_RecentWorkspaces.size() ) return m_RecentWorkspaces[ 0 ]; }
```

Comment:
And if m_RecentWorkspaces is empty? I think this getter is unnecessary. The caller can just call .front() on the result vector from GetRecentWorkspaces()